screens/ActivitiesScreen.tsx

```diff
@@ -30,31 +29,13 @@ const ActivitiesScreen = () => {
 
   return (
     <GestureHandlerRootView style={{ flex: 1 }}>
-      <TouchableOpacity onPress={signOut}>
-        <Text
-          style={{
-            fontSize: 15,
-            margin: 20
-          }}
-        >
-          Log Out
-        </Text>
-        <Text
-          style={{
-            alignItems: "center",
-            justifyContent: "center",
-            color: "black",
-            fontWeight: "bold",
-            fontSize: 15
-          }}
-        >
-          SandBox to get started
-        </Text>
-      </TouchableOpacity>
-      <HexColorPicker
-        color={color}
-        onChange={setColor}
-        options={EventColors.all}
+      <MapComponent
+        containStyle={mapCompStyle.container}
+        mapStyle={mapCompStyle.map}
+        initialRegion={state.initialRegion}
+        markers={state.markers}
+        extractKey={(event) => event.key}
+        movementSettings={state.movementSettings}
       />
     </GestureHandlerRootView>
```

Comment:
I know that storybook for some reason was not liking the map, but maybe what you can do instead of dumping everything in the activities screen (which should probably be removed tbh since we have storybook) you could make a `MapPreview` component so that other's who need to mess around with the map can just use that.

Reply:
I’ll try and get it to work with storybook so we won’t need to do that